scripts/README.md

```diff
@@ -0,0 +1,51 @@
+# Scripts
+
+## Benchmarking on sift1M
+
+### Download dataset
+
+```shell
+wget http://ann-benchmarks.com/sift-128-euclidean.hdf5
+```
+
+### Run storage backends
+
+```shell
+docker-compose up -d
+```
+
+### Run benchmarking script
+
+Install dependencies:
+
+```
+pip install "docarray[common,benchmark,qdrant,annlite,weaviate,elasticsearch,redis]"
```

Comment:
why is common needed here ?

Reply:
`matplotlib` is needed to generate the figure

Reply:
I would suggest adding all to the `benchmark` part

Reply:
yes at least we shouold remove common